src/app/shared/image-provider.service.ts

```diff
@@ -49,10 +52,28 @@ export class ImageProviderService {
     tool.imgProvider = friendlyRegistryName;
     if (registry) {
       tool.imgProviderUrl = this.getImageProviderUrl(tool.path, registry);
+      tool.imgProviderIcon = this.getImageProviderIcon(tool.registry);
     }
     return tool;
   }
 
+  private getImageProviderIcon(registry: RegistryEnum): IconDefinition {
+    if (!registry) {
+      return null;
+    }
+    switch (registry) {
+      case RegistryEnum.QUAYIO:
+        return faQuay;
+      case RegistryEnum.DOCKERHUB:
+        return faDocker;
+      case RegistryEnum.GITLAB:
+        return faGitlab;
+      case RegistryEnum.SEVENBRIDGES: // TODO: Get SevenBridges Icon
+      case RegistryEnum.AMAZONECR: // TODO: Get AMAZON_ECR Icon
```

Comment:
Are there issues for these?

Reply:
In particular, the ECR icon is pretty easy to get from https://aws.amazon.com/architecture/icons/

Reply:
Ive tried looking through the downloaded folder for the ECR icon that Amazon provided, but I cannot find the icon.

Reply:
I do see it in there; look for the file Amazon-EC2-Container-Registry_Image*